package.json

```diff
@@ -2,47 +2,35 @@
   "name": "fetch-paginate",
   "description": "Get multiple pages from paginated APIs with fetch.",
   "version": "6.1.0",
-  "main": "dist/main.js",
-  "types": "dist/index.d.ts",
+  "type": "module",
+  "exports": {
+    ".": {
+      "default": "./dist/index.js",
+      "types": "./dist/index.d.ts"
+    }
+  },
+  "main": "dist/index.js",
   "license": "MIT",
   "repository": "AndersDJohnson/fetch-paginate",
   "funding": "https://github.com/sponsors/AndersDJohnson",
-  "browserslist": [
-    "> .5%",
-    "last 2 versions"
-  ],
   "scripts": {
     "test": "jest",
-    "build": "webpack && tsc",
+    "build": "tsc",
```

Comment:
Hey @dword-design thanks for the contribution! We're able to remove Webpack because we no longer need to transpile now that we're on ESM?